bbp/nse/equation/default.nix

```diff
@@ -0,0 +1,21 @@
+{ fetchurl, pythonPackages, buildPythonPackage }:
+
+buildPythonPackage rec {
+  version = "1.2.01";
+  name = "Equation-${version}";
+
+  src = fetchurl {
+    url = "https://github.com/alphaomega-technology/Equation/releases/download/v${version}/${name}.tar.gz";
```

Comment:
For github, I strongly recommend you to use the function "fetchFromGitHub" which is specially done for it.

The main reason is that github generate tarball on the fly, and consequently they never have the same SHA256 if you simply use "fetchurl"